src/pages/HomePage.tsx

```diff
@@ -28,9 +52,15 @@ const HomePage: React.FC = () => {
 
         {/* Song will hide and show depending on the songVisibiliy boolean which is changed by the button click*/}
         {songVisibility ? (
-          <div id="songDiv">
-            <SongViewer songNumber={number} />
-          </div>
+          lyricsOnlyMode ? (
+            <div id="lyricDiv">
+              <LyricViewer songNumber={number} />
+            </div>
+          ) : (
+            <div id="songDiv">
+              <SongViewer songNumber={number} />
+            </div>
+          )
```

Comment:
I've been using ternary operators to conditionally show and hide different components, this is probably going to get ugly fast if a couple more are added like this.

@slowhei any suggestions?

Reply:
Could you do something like

`songVisibility && lyricsOnlyMode ? (...`

Reply:
Maybe we can have them as separate IonContents and show 1 and hide the other at all times

Reply:
I'll give that a try